hudi-common/src/test/java/org/apache/hudi/common/testutils/FileCreateUtils.java

```diff
@@ -414,9 +415,16 @@ public static String createLogFile(HoodieTableMetaClient metaClient, String part
     if (Files.notExists(logFilePath)) {
       Files.createFile(logFilePath);
     }
-    RandomAccessFile raf = new RandomAccessFile(logFilePath.toFile(), "rw");
-    raf.setLength(length);
-    raf.close();
+    try (RandomAccessFile raf = new RandomAccessFile(logFilePath.toFile(), "rw")) {
+      // Use provided length if >= 6
+      raf.setLength(Math.max(length, 6)); // Set minimum length to 6 bytes
```

Comment:
we better use `HoodieLogFormat.MAGIC.length` instead of constant 6, and should we use the log format writer to add a full header instead of just a `MAGIC` string.